matching-notification-api/src/main/java/org/phenotips/matchingnotification/internal/DefaultMatchingNotificationManager.java

```diff
@@ -205,85 +201,57 @@ public boolean saveOutgoingMatches(List<? extends PatientSimilarityView> similar
     }
 
     @Override
-    public boolean setStatus(Set<Long> matchesIds, String status)
+    public PatientMatch setStatus(Long matchId, String status)
```

Comment:
Since a try-catch was removed, this method can now throw - either some unknown exception which can always happen, or (more likely) an AccessControlExfception. I would document that explicitly